mailroom.go

```diff
@@ -8,6 +8,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/appleboy/go-fcm"
```

Comment:
I think we can remove this package entirely and rely on the firebase admin package only

Reply:
You can do that afterwards